package.json

```diff
@@ -21,12 +21,12 @@
     "build": "deno2node tsconfig.json"
   },
   "devDependencies": {
-    "@grammyjs/types": "^2.9.1",
+    "@grammyjs/types": "^2.11.0",
     "@tsconfig/node16": "^1.0.2",
     "@types/node": "^16.6.1",
-    "deno2node": "^1.4.0",
-    "grammy": "^1.11.2",
-    "typescript": "^4.5.5"
+    "deno2node": "^1.7.1",
+    "grammy": "^1.13.0",
+    "typescript": "^4.9.4"
```

Comment:
What is TS used for?

Reply:
For filling up the disk space of contributors!

Reply:
Great idea!
```suggestion
    "typescript": "^4.9.4",
    "lodash": ">=1.0.0"
```

Reply:
Uhm so you're going to remove it?

Reply:
*sweating profusely, and looking around nervously*

(Yes)

Reply:
Haha nice :D

There's no hurry, I just wasn't sure about the plan lol